### ISSUE TITLE
Add batch normalization to TF models

### ISSUE THREAD

**Issue by ajratner** [HazyResearch/snorkel#532]

**Comment by henryre** [comment]
Notes: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py

Will add to `reLSTM`, not simple linear models.
